Deployment.yaml

```
@@ -0,0 +1,92 @@
+# deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: module-14
+spec:
+  selector:
+    matchLabels:
+      app: module-14
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: module-14
+    spec:
+      containers:
+        - name: todo-app
+          image: todo-app:prod
+          imagePullPolicy: Never
+          ports:
+          - containerPort: 5000
+          env:
+#          envFrom:
+#          - secretRef:
+#              name: app-secret5
+          - name: DB_NAME  
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: DB_NAME  
+          - name: COLLECTION_NAME 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: COLLECTION_NAME 
+          - name: CLIENTID 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: CLIENTID 
+          - name: CLIENTSECRET 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: CLIENTSECRET 
+          - name: LOGGLY_TOKEN 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: LOGGLY_TOKEN
```

Your app treats this as an optional secret, but Minikube doesn't: it could be nice to flag it as optional here:
```suggestion
                key: LOGGLY_TOKEN
                optional: true
```